src/components/NcSelect/NcSelect.vue

```diff
@@ -1086,12 +1086,15 @@ body {
 	}
 
 	&.vs--open .vs__dropdown-toggle {
-		border-color: var(--color-primary-element);
+		outline: 2px solid var(--color-main-background);
+		border-color: var(--color-main-text);
 		border-bottom-color: transparent;
 	}
 
 	&:not(.vs--disabled, .vs--open) .vs__dropdown-toggle:hover {
-		border-color: var(--color-primary-element);
+		outline: 2px solid var(--color-main-text);
+		border-color: var(--color-main-background);
+		outline-offset: -2px;
```

Comment:
I think we should switch the colors here
```suggestion
		outline: 2px solid var(--color-main-background);
		border-color: var(--color-main-text);
```
Otherwise, the `NcSelect` will appear to resize when hovering and return to initial size on opening:

Before:

https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/a30bf155-25e7-4307-84a4-abb089f94f52

After:

https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/787efbcd-0d16-436f-950c-221f662c2c0e